pydantic/_internal/_generate_schema.py

```diff
@@ -125,9 +125,8 @@ def check_validator_fields_against_field_name(
     Returns:
         `True` if field name is in validator fields, `False` otherwise.
     """
-    if isinstance(info, (ValidatorDecoratorInfo, FieldValidatorDecoratorInfo)):
```

Comment:
Should we just add the serialization decorator info instances here instead of removing the check entirely? Same for below...

Reply:
i don't think so because the info is of either of the 3 types, so the check would be redundant